sqlstore/src/main/resources/db/migration/V8801__Default_Targeted_Sequencing.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE Project ADD COLUMN `targetedSequencingId` BIGINT (20) DEFAULT NULL AFTER referenceGenomeId;
+
+ALTER TABLE Project ADD FOREIGN KEY (targetedSequencingId) REFERENCES TargetedSequencing (targetedSequencingId);
```

Comment:
Syntax is `ALTER TABLE Project ADD CONSTRAINT named_constraint FOREIGN KEY (targetedSequencingId)...`